includes/Modules/Sign_In_With_Google.php

```diff
@@ -613,7 +613,7 @@ private function render_disconnect_profile( WP_User $user ) {
 	<h2>
 		<?php
 		/* translators: %s: Sign in with Google service name */
-		esc_html( sprintf( __( '%s via Site Kit by Google', 'google-site-kit' ), _x( 'Sign in with Google', 'Service name', 'google-site-kit' ) ) );
+		echo esc_html( sprintf( __( '%s (via Site Kit by Google)', 'google-site-kit' ), _x( 'Sign in with Google', 'Service name', 'google-site-kit' ) ) );
```

Comment:
Should we extract `Site Kit by Google` to a placeholder as well? I'd think we shouldn't include that in the translatable string.

Reply:
Sounds good. I used `__()` instead of `_x()` because there are other instances of `__( 'Site Kit by Google' )` in the plugin so I didn't want to change a bunch of other files/introduce a new string.